client/gutenberg/extensions/publicize/store/effects.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { getFetchConnectionsPath } from './utils';
+import { setConnections } from './actions';
+
+/**
+ * Effect handler which will refresh the connections for a post.
+ * Internally, it will trigger a fetch the post's connections, and after they
```

Comment:
I think there's a `for` missing: `a fetch for the post's connections`

Reply:
Nice catch - done in f7ded3cd9d801e9ee13b396cba5eae1209f0b4fa.